modules/virt-configuring-masquerade-mode-dual-stack.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc
+
+[id="virt-configuring-masquerade-mode-dual-stack_{context}"]
+= Configuring masquerade mode with dual-stack (IPv4 and IPv6)
+
+You can configure a new virtual machine to use both IPv6 and IPv4 on the default pod network by using cloud-init.
+
+The IPv6 network address must be statically set to `fd10:0:2::2/120` with a default gateway of `fd10:0:2::1` in the virtual machine configuration. These are used by the virt-launcher pod to route IPv6 traffic to the virtual machine and are not used externally. 
+
+Once configured, incoming and outgoing traffic for the virtual machine is routed to both the IPv4 address and the unique IPv6 address of the virt-launcher pod.
+The virt-launcher pod then routes the IPv4 traffic to the DHCP address of the virtual machine, and the IPv6 traffic to the statically set IPv6 address of the virtual machine.
+
+.Prerequisites
+
+* The underlying network for the cluster must be connected with IPv6.
+
+.Procedure
+
+. In a new virtual machine configuration, include an interface with `masquerade` and configure the IPv6 address and default gateway by using cloud-init.
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1alpha3
```

Comment:
Didn't the API gratuate to `v1` by now ?

Reply:
Good question. It is still 'kubevirt.io/v1alpha3' in all the cnv docs on main/enterprise-4.8 branches and I can't see a docs story for updating that for 4.8. 
@phoracek, do you know?

Reply:
That's odd. I'm pretty sure Miguel is right and v1 is available for a while now. v1alpha3 should be still accepted, but it is odd that we never updated the docs. @stu-gott is that something we plan to do?

Reply:
I guess that never the less, the "right" thing to do is be consistent.

A follow up PR can bump it to the correct api version in all docs.

Reply:
Yes, v1 is out now. Good catch. Concur that a separate PR is in order.

Reply:
Aha - @stu-gott is there a docs story for the API bump that I might've missed?

Reply:
Found out via email that API update is not scheduled for 4.8 so leaving as is and this will be updated when it hits downstream.